airesources/Dart/lib/src/logger.dart

```diff
@@ -0,0 +1,13 @@
+part of hlt;
+
+
+class Logger {
+  String _fileName;
+
+  Logger._();
+
+  void call(String log) {
+    if (_fileName == null) return;
+    new File(_fileName).writeAsStringSync("$log\n", mode: FileMode.WRITE_ONLY_APPEND);
```

Comment:
Let's use WRITE_ONLY mode. Not overwriting log by default would be confusing. All other kits overwrite.